src/Resources/skeleton/registration/twig_email.tpl.php

```diff
@@ -3,7 +3,7 @@
 <p>
     Please confirm your email address by clicking the following link: <br><br>
     <a href="{{ signedUrl|raw }}">Confirm my Email</a>.
-    This link will expire in {{ expiresAt|date('g') }} hour(s).
+    This link will expire in {{ expiresAtMessageKey|trans(expiresAtMessageData, 'VerifyEmailBundle') }}.
```

Comment:
iirc, we don't require the bundle to be installed before running this command. And so, the best we could do is detect if verifier is installed AND if it's out-of-date, and throw an exception to tell the user to upgrade. It's probably an edge-case though...

Reply:
done